PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -46,52 +46,69 @@ void setLabelHistoEvSel(Histo& hCollisions)
   hCollisions->GetXaxis()->SetBinLabel(ValuesEvSel::All + 1, "All collisions");
   hCollisions->GetXaxis()->SetBinLabel(ValuesEvSel::Cent + 1, "Centrality ok");
   hCollisions->GetXaxis()->SetBinLabel(ValuesEvSel::CentSel8 + 1, "Centrality + sel8 ok");
-  hCollisions->GetXaxis()->SetBinLabel(ValuesEvSel::CentSel8PosZ + 1, "Centrality + sel8 + posZ ok");
-  hCollisions->GetXaxis()->SetBinLabel(ValuesEvSel::CentSel8PosZTFBorder + 1, "Centrality + sel8 + posZ + TF border ok");
+  hCollisions->GetXaxis()->SetBinLabel(ValuesEvSel::CentSel8TFBorder + 1, "Centrality + sel8 + TF border ok");
+  hCollisions->GetXaxis()->SetBinLabel(ValuesEvSel::CentSel8TFBorderPvContr + 1, "Centrality + sel8 + TF border + PV contr ok");
+  hCollisions->GetXaxis()->SetBinLabel(ValuesEvSel::CentSel8TFBorderPvContrChi2 + 1, "Centrality + sel8 + TF border + PV contr + chi2 ok");
+  hCollisions->GetXaxis()->SetBinLabel(ValuesEvSel::CentSel8TFBorderPvContrChi2PosZ + 1, "Centrality + sel8 + TF border + PV contr + chi2 + posZ ok");
```

Comment:
The bin labels can just mention the last passed check, because (if the `EventRejection` order matches the `TESTBIT` order) it implies that the event passed all the previous checks.

Reply:
Done